### ISSUE TITLE
Nothing happens when I press on the Install Button

### ISSUE THREAD

**Issue by devilAPI** [v-briese/addon-installer#2]
No console output too, i'm on Discord Stable with latest Vizality.

**Comment by valentinegb** [comment]
Yeah... I would expect something to break, I haven't worked on this plugin for quite a few months. When the Vizality rewrite is released, then I'll remake this plugin. (It might even be a part of Vizality by default)

**Comment by devilAPI** [comment]
ok great
